app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -1,5 +1,34 @@
 /* list
 ---------------------------------------------------------------------------- */
+.list-group {
+  &.list-group-settings_list {
+    margin-bottom: 0;
+
+    .list-group-item {
+      border-radius: 0;
+      border: 0;
+      border-left: 5px solid transparent;
+      color: $color-forever_blue;
```

Comment:
In the @mdo code guide declaration order example, color is included as a typographic style and so it should precede visual styles like border-, et al 

http://codeguide.co/#css-declaration-order